rules/false_positives/vitess.yara

```diff
@@ -0,0 +1,13 @@
+rule vitess : override {
+  meta:
+    linux_multi_persist = "high"
```

Comment:
medium

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).